.changeset/fair-beans-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-studio': minor
```

Comment:
Let's add `@finos/legend-graph: minor` to this one and remove it in the other changeset.

Reply:
done